rewrite_id_vue.py

```
@@ -0,0 +1,223 @@
+    'stage': extract_card(content, 'ステージ申請'),
+    'stage_opt': extract_card(content, 'ステージオプション'),
+    'employee': extract_card(content, '従業員申請'),
+    'food': extract_card(content, '販売品・購入品・調理工程申請'),
+    'pr': extract_card(content, 'PR情報'),
+    'announcement': extract_card(content, 'アナウンス'),
+    'map': extract_card(content, '模擬店平面図'),
+    'fire': extract_card(content, '火気使用申請'),
+}
+
+# Remove all cards from content
+for key, title in [('place', '会場申請'), ('power', '消費電力申請'), ('rental', '物品申請'), 
```

Ruff (0.15.18)

[warning] Loop control variable `key` not used within loop body

Rename unused `key` to `_key`

(B007)